app/views/songs/new.html.erb

```diff
@@ -3,5 +3,5 @@
   <h1>Upload a Song</h1>
   <hr>
 
-  <%= render partial: "form" %>
```

Comment:
Why not partial? I mean it still looks like a partial with the _ in front of the file.